deploy/packages/python-packages.yaml

```diff
@@ -139,7 +139,7 @@ packages:
       - python
     specs:
       - py-numpy
-      - py-scipy
+      - py-scipy%gcc
```

Comment:
Uhm… `%gcc` is already included in `target_matrix`

Reply:
True true.. I will revert the change